gulp-tasks/test.js

```diff
@@ -502,6 +503,19 @@ function testMarkdown(filename, contents, options) {
       });
     }
 
+    // Validate listed components
+    do {
```

Comment:
I tried to do that, but the proposed syntax for the blink components is multi-line. That forces me into doing multiple calls to regex.exec().

So, leave as is or change the syntax.